backend/authentication/models.py

```diff
@@ -33,8 +33,6 @@ def __str__(self):
 class Support(models.Model):
     supporter_type = models.ForeignKey("SupportEntityType", on_delete=models.CASCADE)
     supporter_entity = models.IntegerField(null=True)
-    supported_type = models.ForeignKey("SupportEntityType", on_delete=models.CASCADE)
```

Comment:
These two lines shouldn't be removed, right? Maybe the way that this table is set up is flawed, but we have `supporter` and `supported`. The type is either org or user, and the entity would be the corresponding id so that we can keep things as integers throughout.

Reply:
Thanks for the review @andrewtavis. In that case we have to supply a kwarg `related_name` to resolve the issue.

Reply:
Ah so it's problematic in so far as they're both referencing the same foreign key, @to-sta? Makes sense. If we can make that change we should be good here 😊

Reply:
Checks are looking great, btw! Looking forward to finishing up the TS fixes so we're back to all green ✅

Reply:
For this @to-sta, we'd have `related_name="supporter"` and `related_name="supported"` respectively? Just checking the docs a bit and that's what first came to mind :)

Reply:
Makes sense :). I will commit the change now.